scripts/globals/items/dusty_scroll_of_reraise.lua

```diff
@@ -0,0 +1,19 @@
+-----------------------------------------
+-- ID: 5436
+--  Dusty Reraise
+--  grants Reraise 3
+-----------------------------------------
+require("scripts/globals/status")
+require("scripts/globals/msg")
+
+function onItemCheck(target)
+    return 0
+end
+
+function onItemUse(target)
+    if target:hasStatusEffect(dsp.effect.RERAISE) then
+        target:delStatusEffect(dsp.effect.RERAISE)
+        target:addStatusEffect(dsp.effect.RERAISE,3,0,1800)
```

Comment:
I hate pointing this out, but when you removed the `else` you forgot to remove the redundant `addStatus` on line 16! (You'll be adding the status regardless on line 18).